bindings/c/test/mako/utils.c

```diff
@@ -77,3 +77,55 @@ void genkey(char* str, int num, int rows, int len) {
 	}
 	str[len - 1] = '\0';
 }
+
+#if 0
```

Comment:
Can you add one line comment here to clarify this is an alternative sorting algorithm for latency stats computation, where we are currently using `qsort`?